doc/graphColoring/pgr_sequentialVertexColoring.rst

```diff
@@ -37,38 +36,35 @@ the Sequential Vertex Coloring algorithm implemented by Boost.Graph.
 Description
 -------------------------------------------------------------------------------
 
-Sequential Vertex Coloring algorithm is a graph coloring or labeling algorithm
-in which colors or labels are assigned to the vertices of a graph sequentially,
-such that no edge connects two identically labeled or colored vertices.
-
-Efficient graph coloring is an NP-Hard problem, and therefore, the Sequential
-Vertex Coloring algorithm does not always produce optimal coloring. This algorithm
-follows a greedy strategy by iterating through all the vertices sequentially,
-and assigning the smallest possible color that is not used by its neighbors, to
-each vertex.
+Sequential Vertex Coloring algorithm is a graph coloring algorithm in which color
+identifiers are assigned to the vertices of a graph in a sequential manner,
+such that no edge connects two identically colored vertices.
 
 **The main Characteristics are:**
 
 - The implementation is applicable only for **undirected** graphs.
-- Provides the color to be assigned to all the nodes present in the graph.
-- The minimum value of color is 1.
-- The maximum value of color will not be greater than the number of vertices
-  in the graph.
-- Higher color values are used only if required, i.e. the algorithm tries to
-  assign the least possible color to every vertex.
-- The returned rows are ordered in ascending order of the node value.
-- Sequential Vertex Coloring Running Time: :math:`O(|V|*(|d| + |k|))`, where
-  :math:`|V|` is the number of vertices, :math:`|d|` is the maximum degree of
-  the vertices in the graph, and :math:`|k|` is the number of colors used.
+- Provides the color to be assigned to all the vertices present in the graph.
+- Color identifiers values are in the Range :math:`[1, |V|]`
+- The algorithm tries to assign the least possible color to every vertex.
+- Efficient graph coloring is an NP-Hard problem, and therefore, this algorithm
+  does not always produce optimal coloring. It follows a greedy strategy by
+  iterating through all the vertices sequentially, and assigning the smallest
+  possible color that is not used by its neighbors, to each vertex.
+- The returned rows are ordered in ascending order of the vertex value.
+- Sequential Vertex Coloring Running Time: :math:`O(|V|*(|d| + |k|))`
+
+  - where :math:`|V|` is the number of vertices,
+  - :math:`|d|` is the maximum degree of the vertices in the graph,
+  - :math:`|k|` is the number of colors used.
```

Comment:
* I was looking at other functions and we are using `V` instead of `|V|`, so please open an issue in gsoc repo, so that the other functions documentation change to `|V|` and `|E|`  (the `| |` denote size, and without it is a set.
* |d| and |k| remove the bars